src/Shared/FrameworkLocationHelper.cs

```diff
@@ -1188,6 +1180,26 @@ private static void RedirectVersionsIfNecessary(ref Version dotNetFrameworkVersi
             }
         }
 
+#if FEATURE_SYSTEM_CONFIGURATION
```

Comment:
Had to `#if` this out because of lack of System.Configuration--but all of its callsites were already out.